test/e2e/performanceprofile/functests/utils/cgroup/cgroup.go

```diff
@@ -51,3 +54,26 @@ func BuildGetter(ctx context.Context, c client.Client, k8sClient *kubernetes.Cli
 		return cgroupv1.NewManager(c, k8sClient), nil
 	}
 }
+
+// PidCgroupParser parsing /proc/pid/cgroup
+func PidCgroupParser(lines []byte) (string, error) {
```

Comment:
My only suggestion would be to change the name to `PidParser` and outside the package it will be called as followed:
`cgroup.PidParser`

Reply:
fixed in the latest commit